superset-frontend/src/dashboard/components/OverwriteConfirm/OverwriteConfirmModal.tsx

```diff
@@ -0,0 +1,212 @@
+   `}
+`;
+
+const StyledEditor = styled.div`
+  ${({ theme }) => `
+     table {
+       border: 1px ${theme.colors.grayscale.light2} solid;
+     }
+     pre {
+       font-size: 11px;
+       padding: 0px;
+       background-color: transparent;
+       border: 0px;
+       line-height: 110%;
+     }
+   `}
+`;
+
+const StackableHeader = styled(Button)<{ top: number }>`
+  ${({ theme, top }) => `
+     position: sticky;
+     top: ${top}px;
+     background-color: ${theme.colors.grayscale.light5};
+     margin: 0px;
+     padding: 8px 4px;
+     z-index: 1;
+     border: 0px;
+     border-radius: 0px;
+     width: 100%;
+     justify-content: flex-start;
+     border-bottom: 1px ${theme.colors.grayscale.light1} solid;
+     &::before {
+       display: inline-block;
+       position: relative;
+       opacity: 1;
+       content: "\\00BB";
+     }
+   `}
+`;
+
+const StyledBottom = styled.div<{ inView: boolean }>`
+  ${({ inView }) => `
+     margin: 8px auto;
+     text-align: center;
+     opacity: ${inView ? 0 : 1};
+  `}
+`;
+
+type Props = {
+  overwriteConfirmMetadata: DashboardState['overwriteConfirmMetadata'];
+};
+
+const OverrideConfirmModal = ({ overwriteConfirmMetadata }: Props) => {
+  const [bottomRef, hasReviewed] = useInView({ triggerOnce: true });
+  const dispatch = useDispatch();
+  const onHide = useCallback(
+    () => dispatch(setOverrideConfirm(undefined)),
+    [dispatch],
+  );
+  const anchors = useMemo<RefObject<HTMLDivElement>[]>(
+    () =>
+      overwriteConfirmMetadata
+        ? overwriteConfirmMetadata.overwriteConfirmItems.map(() =>
+            createRef<HTMLDivElement>(),
+          )
+        : [],
+    [overwriteConfirmMetadata],
+  );
+  const onAnchorClicked = useCallback(
+    (index: number) => {
+      anchors[index]?.current?.scrollIntoView({ behavior: 'smooth' });
+    },
+    [anchors],
+  );
+  const onConfirmOverwrite = useCallback(() => {
+    if (overwriteConfirmMetadata) {
+      dispatch(
+        saveDashboardRequest(
+          overwriteConfirmMetadata.data,
+          overwriteConfirmMetadata.dashboardId,
+          SAVE_TYPE_OVERWRITE_CONFIRMED,
+        ),
+      );
+    }
+  }, [dispatch, overwriteConfirmMetadata]);
+
+  return (
+    <Modal
+      width="auto"
+      height="50vh"
+      show={Boolean(overwriteConfirmMetadata)}
+      title={t('Confirm overwrite')}
+      footer={
+        <>
+          {t('* Scroll down to the bottom to complete the review. ')}
+          <Button
+            htmlType="button"
+            buttonSize="small"
+            onClick={onHide}
+            data-test="override-confirm-modal-cancel-button"
+            cta
+          >
+            {t('No')}
+          </Button>
+          <Button
+            data-test="overwrite-confirm-save-button"
+            htmlType="button"
+            buttonSize="small"
+            cta
+            buttonStyle="primary"
+            onClick={onConfirmOverwrite}
+            disabled={!hasReviewed}
```

Comment:
Add a tooltip to explain the disabled state?

Reply:
The following description should be enough to explain the disabled state.

<img width="1018" alt="my_new_dashboard" src="https://user-images.githubusercontent.com/1392866/196555634-ee44c4cd-08e6-4536-adb2-538fd028bc36.png">